inst/extdata/database/tedfs/Tech/Electric Arc Furnace.csv

```diff
@@ -1,17 +1,17 @@
 mode,reheating,variable,reference_variable,period,component,value,uncertainty,unit,reference_value,reference_unit,comment,source,source_detail
-*,*,CAPEX,Output Capacity|Steel Liquid,2013,#,184.0,,EUR_2013,1.0,t/a,,Wortler13,
-*,*,CAPEX,Output Capacity|Steel Liquid,2018,#,184.0,,EUR_2013,1.0,t/a,"This reference assumes the value provided by Wörtler et al., 2013.",Vogl18,
+*,*,CAPEX,Output Capacity|Steel Liquid,2013,#,184.0,,EUR_2010,1.0,t/a,,Wortler13,Exhibit 8
+*,*,CAPEX,Output Capacity|Steel Liquid,2018,*,184.0,,EUR_2010,1.0,t/a,"This reference assumes the value provided by Wörtler et al., 2013.",Vogl18,
 *,*,CAPEX,Output Capacity|Steel Liquid,2021,#,479.89,,MEUR_2020,2.0,Mt/a,,Jacobasch21,"Page 9, Table 6"
-*,*,CAPEX,Output Capacity|Steel Liquid,2022,#,331.2,,EUR_2013,1.0,t/a,"This reference assumes the value provided by Wörtler et al., 2013 but with an additional integration factor of 1.8.",AgoraEnergiewende22,
-*,*,OPEX Fixed Relative,,2022,#,3.0,,percent,,,,AgoraEnergiewende22,
+*,*,CAPEX,Output Capacity|Steel Liquid,2022,#,331.2,,EUR_2010,1.0,t/a,"This reference assumes the value provided by Wörtler et al., 2013 (184 EUR/tLS) but with an additional integration factor of 1.8.",AgoraEnergiewende22,Abbildung 27 (Figure 27)
+*,*,OPEX Fixed Relative,,2022,#,3.0,,percent,,,"This entry is FALSE and should be removed. 3% is the OPEX value for BF-BOF, no value is given for EAF from this source.",AgoraEnergiewende22,"Found in the excel tool associated with the report, in Input_Anlage."
 *,*,OPEX Fixed Relative,,2018,Operation and maintenance incl. refractory lining w/o labour,3.0,,percent,,,,Vogl18,Section 2.1
 *,*,OPEX Fixed Specific,Output|Steel Liquid,2018,Labour,53.2,,EUR,1.0,t,,Vogl18,Section 2.1
-*,*,OPEX Fixed Specific,Output|Steel Liquid,2008,#,3.0,,EUR,1.0,t,Labour cost,ECORYS08,Table 3.1
+*,*,OPEX Fixed Specific,Output|Steel Liquid,2008,#,3.0,,EUR,1.0,t,"Labour cost. This line throws an error, as no OCF from this source has been found. Results from this table were obtained from steelonthenet, consultants. No reference to OCF could be found in the report, or on the website of steelonthenet.",ECORYS08,Table 3.1
```

Comment:
Let's simply report this entry as `OPEX Fixed` (i.e. in units of `EUR/a` per `t/a`) assuming a 90% OCF and highlight this assumption in the `comment`. Any chance you could make that adjustment? Thanks!

Reply:
I tried running this, but encounter the same issue as reported in Issue #15

Reply:
It seems to be working for me. Are you sure that you adjusted both variables (`variable` from `OPEX Fixed Specific` to `OPEX Fixed` and `reference_variable` from `Output` to `Output Capacity`) and both units (`unit` from `EUR` to `EUR/a` and `reference_unit` from `t` to `t/a`).

As a side note: I'm just noticing that the reference year for those currencies is missing, e.g. it should be something like `EUR_2008`, not just `EUR`. Any chance you could fix this? Thanks!